packages/tocco-ui/src/Panel/PanelHeaderFooter.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import StyledPanelHeaderFooter from './StyledPanelHeaderFooter'
+import Button from '../Button'
+
+/**
+ * <PanelHeader/> and <PanelFooter/> contain by default a button to toggle the visibility state of <PanelBody>.
+ * Header and footer can contain any content. If both are displayed is up to the implementer.
+ */
+class PanelHeaderFooter extends React.Component {
+  render() {
+    const {
+      // eslint-disable-next-line
+      children,
+      // eslint-disable-next-line
+      isOpen,
+      // eslint-disable-next-line
+      isToggleable,
+      showToggler,
+      // eslint-disable-next-line
+      toggleOpenState
+    } = this.props
+
+    return (
+      <StyledPanelHeaderFooter>
+        <div>
+          {React.Children.map(children, child => React.cloneElement(child))}
+        </div>
+        {isToggleable
+          && showToggler
+          && <Button
+            icon={isOpen ? 'fa-minus' : 'fa-plus'}
+            onClick={toggleOpenState}
+            title={isOpen ? 'Weitere Informationen verbergen' : 'Weitere Informationen anzeigen'}
```

Comment:
Please no hard coded text resources. Use props to inject this texts (as i did with RemoteField for example).
Fallback text can be hardcoded but should be in english (e.g. "More" and "Less")

Reply:
see 88e5dbe45b2124693323e37e12e97f459bcf44ec